pkgs/by-name/e-/e-imzo-manager/package.nix

```
@@ -0,0 +1,87 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  cargo,
+  desktop-file-utils,
+  gnome-desktop,
+  meson,
+  ninja,
+  pkg-config,
+  polkit,
+  rustc,
+  rustPlatform,
+  wrapGAppsHook4,
+  gdk-pixbuf,
+  glib,
+  adwaita-icon-theme,
+  gtk4,
+  libadwaita,
+  openssl,
+  nix-update-script,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "e-imzo-manager";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "xinux-org";
+    repo = "e-imzo";
+    tag = finalAttrs.version;
+    hash = "sha256-uDaqkz2VDvqTgi+k8EGGKjLkjoH93xXHQcgUc1NVo30=";
+  };
+
+  cargoDeps =
+    with finalAttrs;
+    rustPlatform.fetchCargoVendor {
+      inherit pname version src;
+      hash = "sha256-rulWG4L/uN6+JBk+SzC0y57Pdw5N0Q1dJlpXGVo+vbQ=";
+    };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    cargo
+    rustPlatform.cargoSetupHook
+    rustc
+    desktop-file-utils
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    gdk-pixbuf
+    glib
+    gnome-desktop
+    adwaita-icon-theme
+    gtk4
+    libadwaita
+    openssl
+    rustPlatform.bindgenHook
+    polkit
+  ];
+
+  propagatedUserEnvPkgs = [ polkit ];
+
+  postInstall = ''
+    gappsWrapperArgs+=(
+      --prefix PATH : ${lib.makeBinPath finalAttrs.propagatedUserEnvPkgs}
```

```suggestion
      --suffix PATH : ${lib.makeBinPath finalAttrs.propagatedUserEnvPkgs}
```

Just learned this today...  This makes it easier for the user to use a custom version of `polkit` and only use the one provided here as a fallback.

---

Np